msk/actions/create.py

```diff
@@ -194,6 +196,61 @@ def name(self) -> str:
             '-'
         )
     ])
+
+    @Lazy
+    def dependencies(self):
+        deps = "dependencies:\n"
+        if ask_yes_no(message="Does this skill depend on Python Packages (PyPI), System Packages (apt-get/others),"
+                              " or other skills? (y/N)", default=False):
+            if ask_yes_no("Does the skill depend on any Python Packages (PyPI)? (y/N)", default=False):
+                python_deps = ask_input_lines(message="What Python Packages (on PyPI) does your skill need?\n"
+                                                      "(You can include a version number if you need: "
+                                                      "phil >= 1.3.0)", bullet=" - ")
+                deps += """
+    # Pip dependencies on PyPI
+    python:"""
+                for python_dep in python_deps:
+                    deps += "\n      - {}".format(python_dep)
+
+            if ask_yes_no("Does the skill depend on any System Packages (such as apt-get)? (y/N)", default=False):
+                system_deps = ask_input_lines(message="What System Packages does your skill need?\n"
+                                                                     "These Packages will be installed through package"
+                                                                     " managers, like apt-get.", bullet=" - ")
+                deps += """
+    # Install packages with the system package manager
+    # This searches for the provided executable and uses the package names
+    system:
+      all:"""
+                for system_dep in system_deps:
+                    deps += " {}".format(system_dep)
+
+                deps += """
+    # If the package has a certain name on a different platform:
+    # pkcon: pianobar libpiano-dev  # For the mycroft platform
+    # apt-get: pianobar libpiano-dev  # For Ubuntu/Debian
+    #
+    # Require certain executables to be in the PATH for the install to succeed
+#   exes:
+#     - pianobar
+"""
+            if ask_yes_no("Does the skill depend on any other skills that are available through the Skill Marketplace? (y/N)",
+                          default=False):
+                skill_deps = ask_input_lines(message="What skills does your skill depend on?", bullet=" - ")
+                deps += """
+    # Require the installation of other skills before installing this skill
+    skills:
+                """
+                for skill_dep in skill_deps:
+                    deps += "\n      - {}".format(skill_dep)
+
+            ask_yes_no(
+                message="NOTE: It is strongly recommended that you look in at manifest.yml,"
+                        " so you can verify that everying is correct."
+                        "\nThis also lets you set if you want certain executables on your PATH, etc... OKAY? (Y/n)", default=True)
```

Comment:
Wondering if this could be simplified a little eg: 
> NOTE: It is strongly recommended that you check the generated manifest.yml, and verify that the entries are working correctly.
> This also lets you set executables on your PATH (Press enter to continue)"

We need to get some proper documentation on the Skills manifest up and direct people to that really.